guardian/gateway
# show / hide password eye for password reset form

## What does this change?


Implementation of a show / hide password eye for password input fields used on password reset form. 

Implementation details include:
- The eye is only visible if the text field is selected.
- Long password text does not overlap the eye.
- The eye is not present on browsers that already render an icon in the place of the eye.
- prevent update in progress errors on RiffRaff: 9e17724
- fix gateway javascript for ie: https://github.com/guardian/gateway/pull/139/commits/7f1f5b7c2a5233a65ae713364c2a29738ad504a2 https://github.com/guardian/gateway/pull/139/commits/1fc7a95a2d398ba2b93ebd4760ed1a47d99a0650

## How to test


Deploy on code and test on different browsers with browser stack.
Check password reset metrics - metrics should not be affected by this change going out.

## Video



https://user-images.githubusercontent.com/29203769/103409152-4d8c1580-4b5d-11eb-9b00-683477ce3223.mov
